server/scripts/import-tests/index.js

```diff
@@ -141,14 +141,21 @@ const buildTestsAndCreateTestPlanVersions = async (commit, { transaction }) => {
     // Gets the next ID and increments the ID counter in Postgres
     // Needed to create the testIds - see LocationOfDataId.js for more info
     const [testPlanVersionIdResult] = await sequelize.query(
-      `SELECT nextval(
-                pg_get_serial_sequence('"TestPlanVersion"', 'id')
-            )`,
+      `SELECT nextval(pg_get_serial_sequence('"TestPlanVersion"', 'id'))`,
       { transaction }
     );
     const testPlanVersionIdResultRow = testPlanVersionIdResult[0];
     const testPlanVersionId = testPlanVersionIdResultRow.nextval;
 
+    // Get the currently set value to rollback the 'correct' nextval for
+    // subsequent runs
+    const [currentTestPlanVersionIdResult] = await sequelize.query(
+      `SELECT currval(pg_get_serial_sequence('"TestPlanVersion"', 'id'))`,
+      { transaction }
+    );
+    const currentTestPlanVersionId =
+      currentTestPlanVersionIdResult[0].currval - 1;
```

Comment:
Took me a second to understand why `currval -1` works here even with the comment helping explain but I eventually understood and the approach makes sense.